pkg/services/rpcsrv/server.go

```diff
@@ -1719,7 +1718,7 @@ func (s *Server) getStateRoot(ps params.Params) (any, *neorpc.Error) {
 
 func (s *Server) getStorage(ps params.Params) (any, *neorpc.Error) {
 	id, rErr := s.contractIDFromParam(ps.Value(0))
-	if rErr == neorpc.ErrUnknown {
+	if rErr == neorpc.ErrVerificationFailed {
```

Comment:
But why? Shouldn't it be `ErrUnknownContract`? And we have to return it in this case.

Reply:
I haven't yet added this commit to pull request, it's a separate one